apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.py

```diff
@@ -100,6 +100,38 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     )
     element_text = element_inner_html(element)
 
+    submitted_files = data["submitted_answers"].get("_files", [])
+    submitted_file = next(
+        (f for f in submitted_files if f.get("name", None) == file_name), None
+    )
+
+    if data["ai_grading"]:
+        if data["panel"] != "submission" or not submitted_file:
+            return ""
```

Comment:
Another choice would be to include just `<div data-file-name="..."></div>` in this case to represent a placeholder. This would better mirror how our elements end up being rendered in the question panel for AI grading. I actually like that better in general, but I wanted to discuss before changing this.

Reply:
I think this also mirrors better the testing that Victor has done so far, where there is a placeholder of some kind when passing the Question panel to the AI grader.

Reply:
I added such an empty placeholder here in 549ced8.